pkg/mountutil/mountutil.go

```diff
@@ -57,9 +57,12 @@ func ProcessFlagV(s string, volStore volumestore.VolumeStore) (*Processed, error
 	)
 
 	split := strings.Split(s, ":")
-	switch len(split) {
-	case 1:
+
+	if len(split) == 1 || split[0] == "" {
```

Comment:
Why not just remove the leading characters ":" ?

Reply:
Sure, we can do that. I just wanted to isolate under this block. Let me know if you prefer stripping